### PR TITLE
feat(left-nav): add Box Relay icon

### LINKED ISSUE #1206: LeftNav: Add Box Relay icon
Add Box Relay icon to the LeftNav component

### PR DESCRIPTION
**Pull request by sethvoltz**
fixes #1206

### CODE REVIEW

**Comment by boxcla** [comment]
Hi @sethvoltz, thanks for the pull request. Before we can merge it, we need you to sign our Contributor License Agreement. You can do so electronically here: http://opensource.box.com/cla

Once you have signed, just add a comment to this pull request saying, "CLA signed". Thanks!

**Comment by sethvoltz** [comment]
CLA signed

File: src/features/left-sidebar/icons/IconBoxRelay.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    className?: string,
+    color?: string,
+    selected?: boolean,
+    title?: string | React.Element<any>,
+    width?: number,
+};
+
+const iconName = 'icon-box-relay';
+
+const IconBoxRelay = ({ className = '', width = 14, color = '#c4c4c4', selected = false, title }: Props) => (
```
Comment (jstoffan) [inline src/features/left-sidebar/icons/IconBoxRelay.js]: Can this default color come from the variables file?

File: src/features/left-sidebar/icons/IconBoxRelay.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    <AccessibleSVG
+        className={classNames(iconName, className, {
```
Comment (jstoffan) [inline src/features/left-sidebar/icons/IconBoxRelay.js]: Seems like this should be handled by passing in the appropriate `className` prop from the host component.

File: src/features/left-sidebar/icons/IconBoxRelay.js (hunk 1 above)
Comment (sethvoltz) [reply src/features/left-sidebar/icons/IconBoxRelay.js]: This follows the pattern established by the other leftnav icons (see `IconAllFiles`, `IconFavorites`, `IconNotes`). The purpose of this PR is to solve the addition of a new icon, not to refactor the codebase into constants references.

File: src/features/left-sidebar/icons/IconBoxRelay.js (hunk 2 above)
Comment (sethvoltz) [reply src/features/left-sidebar/icons/IconBoxRelay.js]: Again, the purpose of this is to follow the conventions and add a new icon. A refactor seems like a good idea here, however it should be handled as a separate issue ticket and a more thorough review of these icons.

**Pull request merged**